Framework/HistogramData/test/FixedLengthVectorTest.h

```diff
@@ -84,7 +84,6 @@ class FixedLengthVectorTest : public CxxTest::TestSuite {
     FixedLengthVectorTester src(2, 0.1);
     TS_ASSERT_EQUALS(src.size(), 2);
     const FixedLengthVectorTester dest(std::move(src));
-    TS_ASSERT_EQUALS(src.size(), 0);
```

Comment:
Is this now actually doing a copy?

Reply:
No were still moving, but we should not be accessing a moved from item - whilst I seem to recall it's not UB it's still relying on an implementation detail which isn't really documented or expected